### ISSUE TITLE
ImportError: cannot import name 'Timeout'

### ISSUE THREAD

**Issue by oblitum** [rndusr/stig#54]
I've just updated ArchLinux and aiohttp was updated, after that when starting `stig` I get:

```
Traceback (most recent call last):
  File "/usr/bin/stig", line 11, in <module>
    load_entry_point('stig==0.8.1a0', 'console_scripts', 'stig')()
  File "/usr/lib/python3.6/site-packages/stig/__init__.py", line 16, in run
    from . import main
  File "/usr/lib/python3.6/site-packages/stig/main.py", line 27, in <module>
    from . import settings
  File "/usr/lib/python3.6/site-packages/stig/settings/__init__.py", line 13, in <module>
    from .types_srv import is_srv_setting
  File "/usr/lib/python3.6/site-packages/stig/settings/types_srv.py", line 18, in <module>
    from ..client.errors import ClientError
  File "/usr/lib/python3.6/site-packages/stig/client/__init__.py", line 19, in <module>
    from .utils import Response
  File "/usr/lib/python3.6/site-packages/stig/client/utils.py", line 35, in <module>
    from aiohttp import Timeout as AsyncIOTimeout
ImportError: cannot import name 'Timeout'
```

**Comment by rndusr** [comment]
Thanks for your report.

I made a bugfix release.

**Comment by oblitum** [comment]
@rndusr there's no tag for v0.8.2a though.

**Comment by rndusr** [comment]
I wish I knew how to git. Any ideas how to fix this?

I have a bugfix branch based on v0.8.1a that's tagged correctly. I pushed it to
github, but that new tag doesn't show up. The current master branch isn't in a
releasable state, so I can't just tag that. Anyway, it would differ too much
from the bugfix release on pypi.

**Comment by oblitum** [comment]
@rndusr this may be possibly related with how GitHub handle its flag list. I just checked the tags on https://github.com/rndusr/stig/releases and https://github.com/rndusr/stig/tags. There's no tag for v0.8.2a there.

- Do you want to tag just at the bugfix branch? Did you do it already?
- Did you push with tags (`git push origin branch --tags`) ?

**Comment by rndusr** [comment]
Ah, yes, I just forgot to push with --tags. It's there now. Thanks!

**Comment by oblitum** [comment]
Thanks, I can now ask for an update on [AUR](https://aur.archlinux.org/packages/stig).

**Comment by kazuoteramoto** [comment]
Arch Linux packages updated. The fix included in 0.8.2a fixes the problem with aiohttp 3.0.1.

Thanks!

**Comment by oblitum** [comment]
Thanks @kazuoteramoto, that was fast, just updated from all, all working now.
